analyses/driver/matcher.go

```diff
@@ -9,19 +11,64 @@ type RequestType struct {
 	DepsTreePath    string
 }
 
+// StackSummary is SA Result Summary output
+type StackSummary struct {
```

Comment:
Create a separate package for the summay, all the types related to summary must go into the new package

Reply:
Ack.